examples/advanced_usage/handle_cursor_pagination.py

```diff
@@ -0,0 +1,33 @@
+from atproto import Client
+from atproto.xrpc_client import models
+
+
+def main():
+    # This is an example for get_follows method.
+    client = Client()
+    client.login('my-handle', 'my-password')
+    handle = 'target-handle'
+
+    def _fetch(actor: str, cursor: str | None) -> models.AppBskyGraphGetFollows.Response:
+        d = models.AppBskyGraphGetFollows.Params(actor=actor, limit=100)
```

Comment:
could we please call it `params` instead?

Reply:
[bcbbd9c](https://github.com/MarshalX/atproto/pull/93/commits/bcbbd9cde0aa14b7286084d1b75b8c3adf3107de)